ElementX/Sources/Screens/Settings/SettingsScreen/View/SettingsScreen.swift

```diff
@@ -213,6 +220,16 @@ struct SettingsScreen: View {
                 .accessibilityIdentifier(A11yIdentifiers.settingsScreen.done)
         }
     }
+    
+    @ViewBuilder
+    private var secureBackupBadge: some View {
+        if context.viewState.showSecureBackupBadge {
+            BadgeView()
+                .frame(width: 10, height: 10)
+        } else {
+            EmptyView()
+        }
```

Comment:
As you're using a ViewBuilder, the EmptyView is inferred. I think this applies to a few of these.
```suggestion
        }
```

Reply:
Hmm..not entirely sure what I was thinking 😅